tests/src/test/java/alluxio/underfs/hdfs/LocalMiniDFSCluster.java

```diff
@@ -232,7 +232,7 @@ public void start() throws IOException {
 
       // For HDFS of earlier versions, getFileSystem() returns an instance of type
       // {@link org.apache.hadoop.fs.FileSystem} rather than {@link DistributedFileSystem}
-      mDfsClient = (DistributedFileSystem) mDfsCluster.getFileSystem();
+      mDfsClient = mDfsCluster.getFileSystem();
```

Comment:
Here the conversion is intentional and we should keep it. See the comment for the reason.